### ISSUE TITLE
Add Karmilla

### ISSUE THREAD

**Issue by hmaesta** [google/fonts#1542]
![quotation-marks-v011-sample](https://user-images.githubusercontent.com/741969/38965861-8923dc30-4355-11e8-9735-0983b2b2308c.png)

I _really_ like **Karla** ([Google Fonts](https://fonts.google.com/specimen/Karla?selection.family=Karla)), but unfortunately that is a dead project.
**Karmilla** is a replacement, with support for several languages. 

> Karmilla is a slightly modified version of Karla.
>
> Karla is an outstanding grotesque sans serif typeface family created by Jonathan Pinhorn (@jonpinhorn_type, a graduate of the MA in Typeface Design at the University of Reading), released through Google Webfonts in 2012.
> 
> The aim of Karmilla is to offer a workaround for designers who want to use Karla but need **support for French, German, Norvegian, Slovak or Icelandic, which requires adding and tweaking some glyphs.**

→ **https://github.com/ms-studio/karmilla**

**Comment by hmaesta** [comment]
Realized that is already a discussion about this on https://github.com/ms-studio/karmilla/issues/31.

**Comment by davelab6** [comment]
Also mentioned today https://twitter.com/eddiehultin/status/1070331897820127232

**Comment by andrewshadura** [comment]
Any updates please?

**Comment by hmaesta** [comment]
I think this font project is abandoned.

https://github.com/ms-studio/karmilla/issues/54

**Comment by andrewshadura** [comment]
No, it certainly isn’t.

**Comment by RosaWagner** [comment]
Hi @hmaesta, we actually updated Karla's glyphset at the beginning of the year. There are some clear design issues though (capital eszett for example), but we would rather continue improving Karla instead of onboarding a new family. If you are willing to improve Karla, you can tag me in a pull request in https://github.com/googlefonts/karla and I'll be happy to review ;)

**Comment by andrewshadura** [comment]
Well, I asked in https://github.com/googlefonts/karla/issues/8, and received no response for two years…

**Comment by RosaWagner** [comment]
yeah well we don't "watch" all the `googlefonts` repositories (they are hundreds of them and we are 5 people) but if you tag me or another onboarder we can give it a look. I mean for pull requests. In this repository you can also see that we have several thousands of open issues and a queue of hundreds of projects to onboard or debug. It's not that they are dead, it just take time, and I am giving time now.

**Comment by andrewshadura** [comment]
Great, thanks.
I guess the last missing bits are the Cyrillic letters, possibly something else. Once you pull those in, as @ms-studio pointed out, the *raison d'être* for the fork will cease to exist.

**Comment by emmamarichal** [comment]
Hello everyone! I just opened the sources again in fontforge, and I saw that italics are missing for cyrillic (at least in bold).

So in the current state (missing glyphs, old fontforge file, no italics), it's complicated for us to merge Cyrillic with Karla.
Too many elements are still missing. If you want to update this, it would be easier to work directly on the Karla, and on the Glyphs or ufo files.
Thanks anyway for this sollicitation :)
